_posts/2018-08-02-read-only-functions.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Serverless Security&#58; read-only functions with OpenFaaS
+description: Alex Ellis introduces the read-only functions feature for OpenFaaS - a security concept available with containers and Kubernetes to prevent tampering
+date: 2018-08-01
+image: /images/read-only/railings.jpg
+categories:
+  - security
+  - kubernetes
+  - swarm
+author_staff_member: alex
+---
+
+In this post I'll highlight one of the ways we're making OpenFaaS a more secure environment for your production functions and workloads.
+
+## What is a read-only function?
```

Comment:
keyword mutable? or immutable ?

Reply:
> It would be nice to have a why giving an example of why this is something the reader should want in their functions.
 
Thought I'd covered that in detail, but maybe not? Can you check again with latest edits.

Reply:
Its ok as-is 👍